src/isIsogram.test.js

```diff
@@ -6,4 +6,34 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should retutrn 'true' `
+  + `if 'word' is empty 'string'`, () => {
+    expect(isIsogram(''))
+      .toBe(true);
```

Comment:
try to use .toBeTruthy() or .toBeFalsy()

Reply:
fixed